azure-pipelines/official.yml

```diff
@@ -0,0 +1,37 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "5 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+  always: true
+
+stages:
```

Comment:
This is copied from PR job. except: rcache -> wcache
I will template them later. Maybe setup Azure pipeline first.